cmd/commands/add.go

```diff
@@ -41,10 +42,30 @@ var Add = &cli.Command{
 
 // addCmd is the command to add a plugin of sdk
 func addCmd(ctx *cli.Context) error {
-	manager := internal.NewSdkManager()
-	defer manager.Close()
-	sdkName := ctx.Args().First()
-	source := ctx.String("source")
-	alias := ctx.String("alias")
-	return manager.Add(sdkName, source, alias)
+	args := ctx.Args()
+	var errStr = ""
+	var source = ""
+	var alias = ""
+	var err error
+
+	for _, sdkName := range args.Slice() {
+		// only for when adding one plugin
+		if args.Len() == 1 {
+			source = ctx.String("source")
+			alias = ctx.String("alias")
+		}
+
+		manager := internal.NewSdkManager()
```

Comment:
no need to instantiate SdkManager every time in the for loop

Reply:
Thank you very much for your careful reading, let me avoid some common sense mistakes;